src/isIsogram.test.js

```diff
@@ -8,12 +8,22 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    expect(typeof isIsogram('word')).toBe('boolean');
+  });
 
+  it(`should not be case sensitive`, () => {
+    expect(isIsogram('WoRd')).toBeTruthy();
   });
```

Comment:
It doesn't check case sensitivity, you don't have the same letter in different cases (like `'Aa'`)